src/Sylius/Bundle/OrderBundle/phpunit.xml.dist

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/8.5/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
          colors="true"
 >
     <php>
         <server name="KERNEL_CLASS" value="AppKernel" />
     </php>
 
     <testsuites>
-        <testsuite name="SyliusOrderBundle Test Suite">
+        <testsuite name="sylius_order_bundle_test_suite">
```

Comment:
I know I was suggesting something else (`snake_case`) earlier, but ultimately we would use this syntax
```suggestion
        <testsuite name="sylius-order-bundle">
```